components/script/dom/document.rs

```diff
@@ -4479,6 +4499,15 @@ impl DocumentMethods for Document {
     fn ExitFullscreen(&self) -> Rc<Promise> {
         self.exit_fullscreen()
     }
+
+    // Servo only API to get an instance of the controls of a specific
+    // media element matching the given id.
+    fn ServoGetMediaControls(&self, id: DOMString) -> Fallible<DomRoot<ShadowRoot>> {
```

Comment:
Is there a danger of this getting overwritten by client code? We may need something like XOWs to do this right?

(Not that we have to do that in this PR but we should be careful when shipping)

Reply:
I guess client code code could overwrite this. But I cannot think of a way to exploit that in a dangerous way. In any case, I agree that we may need something like XOWs to do this right.